secrets/secrets.yml

```diff
@@ -0,0 +1,221 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Creates the bucket and lambda needed to support application secrets
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Stack Dependencies
+        Parameters:
+          - InfrastructureStorageStackName
+          - NotificationsStackName
+      - Label:
+          default: Infrastructure Config
+        Parameters:
+          - InfrastructureConfigStagingKey
+          - InfrastructureConfigProductionKey
+    ParameterLabels:
+      InfrastructureStorageStackName:
+        default: Storage stack name
+      NotificationsStackName:
+        default: Notifications stack name
+      InfrastructureConfigStagingKey:
+        default: Staging S3 object key
+      InfrastructureConfigProductionKey:
+        default: Production S3 object key
+Parameters:
+  InfrastructureStorageStackName:
+    Default: infrastructure-storage
+    Description: The name of a previously launched infrastructure storage stack
+    Type: String
+  NotificationsStackName:
+    Default: notifications
+    Description: The name of a previously launched notifications stack
+    Type: String
+  InfrastructureConfigStagingKey:
+    Default: template-config-staging.zip
+    Description: >
+      The S3 object key for the staging template configuration file archive
+      (.zip), inside the InfrastructureConfigBucket
+    Type: String
+  InfrastructureConfigProductionKey:
+    Default: template-config-production.zip
+    Description: >
+      The S3 object key for the production template configuration file archive
+      (.zip), inside the InfrastructureConfigBucket
+    Type: String
+Resources:
+  SecretsIAMGroup:
+    Type: "AWS::IAM::Group"
+    Properties:
+      GroupName: !Sub ${InfrastructureStorageStackName}-manage-secrets
+      Path: "/"
+      Policies:
+        - PolicyName: !Sub ${InfrastructureStorageStackName}-encrypt-secrets
```

Comment:
for now I'm going to keep this compatible with the init script, and leave these in